### PR TITLE
Fix ssl package dependency

### PR DESCRIPTION
**Pull request by derschatta**
Builds are failing due to a missing dependency.

```
 > [php-8.0 10/29] RUN wget "http://security.debian.org/debian-security/pool/updates/main/o/openssl/libssl1.0.0_1.0.1t-1+deb8u12_amd64.deb"     && DEBIAN_FRONTEND=noninteractive dpkg -i ./libssl1.0.0_1.0.1t-1+deb8u12_amd64.deb:
#0 0.302 --2023-07-04 01:31:15--  http://security.debian.org/debian-security/pool/updates/main/o/openssl/libssl1.0.0_1.0.1t-1+deb8u12_amd64.deb
#0 0.302 Resolving security.debian.org (security.debian.org)... 151.101.130.132, 151.101.66.132, 151.101.2.132, ...
#0 0.425 Connecting to security.debian.org (security.debian.org)|151.101.130.132|:80... connected.
#0 0.443 HTTP request sent, awaiting response... 404 Not Found
#0 0.747 2023-07-04 01:31:16 ERROR 404: Not Found.
```

I've update the dependency to the latest one.

### CODE REVIEW

File: php/php74/Dockerfile [hunk 1]
```diff
@@ -86,8 +86,7 @@ RUN curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - \
 
 # Workaround to get MSSQL connection working on Debian 9 (stretch)
 # https://emacstragic.net/2017/11/06/mssql-odbc-client-on-debian-9-stretch/
-RUN wget "http://security.debian.org/debian-security/pool/updates/main/o/openssl/libssl1.0.0_1.0.1t-1+deb8u12_amd64.deb" \
-    && DEBIAN_FRONTEND=noninteractive dpkg -i ./libssl1.0.0_1.0.1t-1+deb8u12_amd64.deb
+Dockerfile
```
Comment (scq) [inline php/php74/Dockerfile:89]: typo?

Comment (derschatta) [reply php/php74/Dockerfile:89]: Good catch, will fix it

**Review by scq — CHANGES_REQUESTED** [review]
This is a pretty interesting one.

All of the containers older than PHP 7.2 no longer build, as Debian 9 (stretch) ended support and the packages were moved to the archive. It should be possible to make them build again by updating sources.list to point to the archive, but at the moment, they are not building. Not sure if we want to address this in this ticket or elsewhere.

The revised URL for the libssl .deb is now also 404ing.

It looks like Microsoft actually fixed the issue with the MSSQL driver though, and manually adding that specific version (1.0.0) of libssl is no longer necessary, at least in Debian 10 (can't test with 9 as the containers won't build).

**Comment by derschatta** [comment]
> 

Thanks @scq  I'll have another look

**Comment by derschatta** [comment]
@scq  ok took me a while to get all the builds running again (at least back to 5.6). It's awesome to be able to get rid of that annoying ssl dependency.

Please have another look

**Review by scq — APPROVED** [review]
Woo, nice one! Looks good :D

**Comment by derschatta** [comment]
fixes #237

**Pull request merged**